setup.py

```diff
@@ -83,7 +84,24 @@ def get_extensions():
 
     main_file = glob.glob(os.path.join(extensions_dir, '*.cpp'))
     source_cpu = glob.glob(os.path.join(extensions_dir, 'cpu', '*.cpp'))
-    source_cuda = glob.glob(os.path.join(extensions_dir, 'cuda', '*.cu'))
+
+    is_rocm_pytorch = True if ROCM_HOME is not None else False
```

Comment:
Can we also here use the hip version @t-vi added?

Reply:
Good catch. Added it, now it is a more reliable check